frontend/javascripts/admin/auth/finish_reset_password_view.tsx

```diff
@@ -49,79 +49,81 @@ function FinishResetPasswordView(props: Props) {
 
   return (
     <Row className="login-view" justify="center" align="middle">
-      <Col className="login-content">
-        <h3>Reset Password</h3>
-        <Form onFinish={onFinish} form={form}>
-          <FormItem
-            hasFeedback
-            name={["password", "password1"]}
-            rules={[
-              {
-                required: true,
-                message: messages["auth.reset_new_password"],
-              },
-              {
-                min: 8,
-                message: messages["auth.registration_password_length"],
-              },
-              {
-                validator: (_, value: string) =>
-                  checkPasswordsAreMatching(value, ["password", "password2"]),
-              },
-            ]}
-          >
-            <Password
-              prefix={
-                <LockOutlined
-                  style={{
-                    fontSize: 13,
-                  }}
-                />
-              }
-              placeholder="New Password"
-            />
-          </FormItem>
-          <FormItem
-            hasFeedback
-            name={["password", "password2"]}
-            rules={[
-              {
-                required: true,
-                message: messages["auth.reset_new_password2"],
-              },
-              {
-                min: 8,
-                message: messages["auth.registration_password_length"],
-              },
-              {
-                validator: (_, value: string) =>
-                  checkPasswordsAreMatching(value, ["password", "password1"]),
-              },
-            ]}
-          >
-            <Password
-              prefix={
-                <LockOutlined
-                  style={{
-                    fontSize: 13,
-                  }}
-                />
-              }
-              placeholder="Confirm New Password"
-            />
-          </FormItem>
-          <FormItem>
-            <Button
-              type="primary"
-              htmlType="submit"
-              style={{
-                width: "100%",
-              }}
+      <Col>
+        <Card className="login-content">
```

Comment:
Wrapped view in `<Card>` component in order for the `<FormItem>` / `<Input>` components to have a correctly colored backgrounds in dark mode.

Effectively this colors the login view dark in dark mode.

Reply:
The rest of the component is identical just indented.